app/models.py

```diff
@@ -419,6 +419,12 @@ class Organisation(db.Model):
     billing_contact_email_addresses = db.Column(db.Text, nullable=True)
     billing_reference = db.Column(db.String(255), nullable=True)
 
+    email_branding_pool = db.relationship(
+        'EmailBranding',
+        secondary='email_branding_to_organisation',
+        backref='organisation'
```

Comment:
So an `EmailBranding` can belong to multiple organisations. Therefore I don't think it makes sense for the backref to be `organisation` (singular) as it could return multiple organisations.

I think this should likely mean the backref is renamed to `organisations`.

Note, I've been reading this example to see a generic example for sqlalchemy example of a many to many relationship like this https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#many-to-many

Reply:
We will give this a go, thanks, great suggestion and the docs were really helpful.

Reply:
done now ☺️